synapse/storage/schema/delta/56/fix_room_keys_index.sql

```diff
@@ -0,0 +1,18 @@
+/* Copyright 2019 Matrix.org Foundation CIC
```

Comment:
@richvdh does this file need to be bumped to `.../delta/57/`?

Reply:
no, it's fine to add more deltas to the current directory.